containers/app/Dockerfile

```
@@ -5,7 +5,8 @@ FROM node:24-bookworm-slim@sha256:242549cd46785b480c832479a730f4f2a20865d61ea2e4
 ENV NEXT_TELEMETRY_DISABLED=1
 WORKDIR /workspace
 
-COPY package.json package-lock.json ./
+COPY package.json package-lock.json .npmrc ./
+RUN npm install --global "npm@$(node -p \"require('./package.json').packageManager.slice(4)\")"
```

Hadolint (2.14.0)

[error] Expected end of $(..) expression. Fix any mentioned problems and try again.

(SC1072)